tapir/core/tests/test_email_preferences.py

```diff
@@ -0,0 +1,50 @@
+from tapir.accounts.models import TapirUser
+from tapir.accounts.tests.factories.factories import TapirUserFactory
+from tapir.coop.emails.tapir_account_created_email import TapirAccountCreatedEmail
+from tapir.core.tapir_email_base import TapirEmailBase
+from tapir.utils.tests_utils import LdapEnabledTestCase
+
+
+class VoluntaryMail(TapirEmailBase):
+    mandatory = False
+
+    # def __init__(self, tapir_user: TapirUser):
+    #     super().__init__()
+    #     self.tapir_user = tapir_user
+
+    @classmethod
+    def get_unique_id(cls) -> str:
+        return "tapir.coop.voluntaryTestMail"
+
+    @classmethod
+    def get_name(cls) -> str:
+        return "VoluntaryTestMail"
+
+
+class TestVoluntaryMails(LdapEnabledTestCase):
+    def setUp(self):
+        TapirEmailBase.register_email(VoluntaryMail)
+
+    def test_has_voluntary_mail_NOT_subscribed(self):
+        tapir_user: TapirUser = TapirUserFactory.create()
+
+        assert (
```

Comment:
I usually use `self.assert...()` as they give better messages when the test fails.